editor/plugins/sprite_2d_editor_plugin.h

```diff
@@ -60,14 +63,23 @@ class Sprite2DEditor : public Control {
 	AcceptDialog *err_dialog = nullptr;
 
 	ConfirmationDialog *debug_uv_dialog = nullptr;
-	Control *debug_uv = nullptr;
+	Panel *debug_uv = nullptr;
 	Vector<Vector2> uv_lines;
 	Vector<Vector<Vector2>> outline_lines;
 	Vector<Vector<Vector2>> computed_outline_lines;
 	Vector<Vector2> computed_vertices;
 	Vector<Vector2> computed_uv;
 	Vector<int> computed_indices;
 
+	HScrollBar *h_scroll = nullptr;
+	VScrollBar *v_scroll = nullptr;
+	EditorZoomWidget *zoom_widget = nullptr;
+	Ref<ViewPanner> panner;
+	Vector2 draw_offset;
+	real_t draw_zoom;
+	bool updating_zoom;
+	bool center_on_draw;
```

Comment:
```suggestion
	real_t draw_zoom = 1.0;
	bool updating_zoom = false;
	bool center_on_draw = false;
```
Primitive types should be initialized.
`updating_zoom` could be removed like in #83731

Reply:
And before you notice it, center_on_draw is unused anyway, I'll remove it. It's a remnant from trying to find solution for centering the view upon opening.